test/client/integration.test.ts

```diff
@@ -45,6 +48,21 @@ describe("Client Integration", function() {
         const apiPacker = CoreAPI.new(storagePacker, genesis, provider, signer);
         const apiSyncer = CoreAPI.new(storageSyncer, genesis, provider, signer);
 
+        for (const user of group.userIterator()) {
+            await apiSyncer.contracts.blsAccountRegistry.register(user.pubkey);
+            // TODO Is this the right way/place to do this?
+            const tokenID = 1;
+            const initialBalance = USDT.fromHumanValue("100.12");
+            const state = State.new(
+                user.pubkeyID,
+                tokenID,
+                initialBalance.l2Value,
+                0
+            );
+            await apiSyncer.l2Storage.state.update(user.stateID, state);
+        }
+        await apiSyncer.l2Storage.state.commit();
```

Comment:
@ChihChengLiang This was needed to setup syncer state for transaction below since it was removed from storage factory. Do we already the functionality in the node to setup initial state/balances or is that future work?

Reply:
We do not have deposit syncing yet, used L2 setup